vue-custom-bootstrap/src/views/AssessmentSectionList.vue

```diff
@@ -0,0 +1,30 @@
+<template>
```

Comment:
I'd prefer to use the bootstrap components for this instead of just the css classes: https://bootstrap-vue.js.org/docs/components/nav#vertical-variation

In general with bootstrap-vue we can avoid writing almost all CSS by using the built components combined with the built in classes: https://getbootstrap.com/docs/4.3/utilities/float/


```
<b-list-group vertical ... class="mv-2">
  <b-list-group-item :href="getHref(...)">
  {{ item.title }}
  <b-badge variant="info">
...
```